web/src/main/angular/src/app/core/components/server-map/class/multi-connect-node-group.class.ts

```diff
@@ -37,7 +37,8 @@ export class MultiConnectNodeGroup extends NodeGroup {
             'instanceCount': 0,
             'topCountNodes': [],
             'applicationName': this.applicationName,
-            'mergedSourceNodes': []
+            'mergedSourceNodes': [],
+            'agentResponseStatistics':  {}
```

Comment:
I think there's no need to add `agentResponseStatistics` here. But I could be wrong. Is it necessary?